io-interface/src/ultra_sonic_sensor.cpp

```diff
@@ -0,0 +1,7 @@
+#include <io-interface/ultra_sonic_sensor.hpp>
```

Comment:
Also, since it will be a device specific library, I wonder if we would rather not create a subfolder like: `io-interface/src/raspi/` which will contain the device specific things for the raspi. And therefore, the library would be called something along the line `io-interface-raspi`. What do you think?